malariagen_data/anoph/karyotype.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd  # type: ignore
+from pandas import CategoricalDtype
+import numpy as np  # type: ignore
+import allel  # type: ignore
+
+from numpydoc_decorator import doc
+from ..util import check_types, _karyotype_tags_n_alt
+from . import base_params
+from typing import Optional
+
+from .snp_data import AnophelesSnpData
+from .karyotype_params import inversion_param
+
+
+class AnophelesKaryotypeData(AnophelesSnpData):
```

Comment:
```suggestion
class AnophelesKaryotypeAnalysis(AnophelesSnpData):
```

Nit, suggest slight naming change, then will hopefully also make sense if you later on add in here some functions for calculating inversion allele frequencies.

Reply:
I agree.